test/challtestsrv/challenge-servers.go

```diff
@@ -41,16 +45,18 @@ type Config struct {
 	HTTPOneAddrs []string
 	// DNSOneAddrs are the DNS-01 challenge server bind addresses/ports
 	DNSOneAddrs []string
+	// TLSALPNOneAddrs are the TLS-ALPN-01 challenge server bind addresses/ports
+	TLSALPNOneAddrs []string
 }
 
 // validate checks that a challenge server Config is valid. To be valid it must
 // specify a bind address for at least one challenge type. If there is no
 // configured log in the config a default is provided.
 func (c *Config) validate() error {
 	// There needs to be at least one challenge time with a bind address
```

Comment:
You didn't make the typo but would you mind fixing "challenge time" -> "challenge type" in this branch while you're here?